### PR TITLE
[Android] Fix for back button does not work as expected on SDK 28

### PR DESCRIPTION
**Pull request by MaximeLimone**
Fix for #186

### CODE REVIEW

File: src/android/SplashScreen.java [hunk 1]
```diff
@@ -97,12 +98,14 @@ protected void pluginInitialize() {
         }
         // Make WebView invisible while loading URL
         // CB-11326 Ensure we're calling this on UI thread
-        cordova.getActivity().runOnUiThread(new Runnable() {
-            @Override
-            public void run() {
-                getView().setVisibility(View.INVISIBLE);
-            }
-        });
+        if (Build.VERSION.SDK_INT < 28) {
```
Comment (timbru31) [inline src/android/SplashScreen.java]: please use `android.os.Build.VERSION_CODES.P` instead of `28`.

File: src/android/SplashScreen.java [hunk 2]
```diff
@@ -97,12 +98,14 @@ protected void pluginInitialize() {
...
+               cordova.getActivity().runOnUiThread(new Runnable() {
+                   @Override
+                   public void run() {
+                      getView().setVisibility(View.INVISIBLE);
+              }
```
Comment (timbru31) [inline src/android/SplashScreen.java]: the indentation seems wrong here.

**Comment by freddysu** [comment]
Thanks for fixing the bug! Really need this one. When will this be merge back and release?
Or is there any workaround I can do now? Thank!

**Comment by janpio** [comment]
Before it can be release it has to be changed (see comments) and then merged.

You can always fork a plugin and apply any patches you want to your fork.

**Comment by prageeth** [comment]
Since these changes haven't happened for a while, I've created another PR with the required changes and CR feedback. @timbru31 Can you please review and merge? Thanks https://github.com/apache/cordova-plugin-splashscreen/pull/225

**Comment by NiklasMerz** [comment]
Shoudl be solved with https://github.com/apache/cordova-plugin-splashscreen/commit/66d1e00e003fb2f935cdbb8a5b0e47b416836b68

**Pull request closed without merging**